src/index.tsx

```diff
@@ -319,9 +320,17 @@ export default class Superwall {
     );
   }
 
+  async confirmAllAssignments(): Promise<ConfirmedAssignment[]> {
+    await this.awaitConfig();
+    const assignments = await SuperwallReactNative.confirmAllAssignments();
+    return assignments.map((assignment: any) =>
+      ConfirmedAssignment.fromJson(assignment)
+    );
+  }
+
   async getConfigurationStatus(): Promise<ConfigurationStatus> {
     const configurationStatusString =
-      await SuperwallReactNative.getConfigurationStatus();
+      await SuperwallReactNative.confirmAllAssignments();
```

Comment:
Is this supposed to have changed

Reply:
no idea how that got in, might have accidentally replaced something 🤔 fix pushed